apps/api/src/domain/models/audio-visual/audio-item/commands/translate-audio-item-name/audio-item-name-translated.event-handler.ts

```diff
@@ -0,0 +1,29 @@
+import { MultilingualTextItemRole } from '@coscrad/api-interfaces';
+import { Inject } from '@nestjs/common';
+import { ICoscradEventHandler } from '../../../../../../domain/common';
+import {
+    AUDIO_QUERY_REPOSITORY_TOKEN,
+    IAudioItemQueryRepository,
+} from '../../queries/audio-item-query-repository.interface';
+import { AudioItemNameTranslated } from './audio-item-name-translated-event';
+
+export class AudioItemNameTranslatedEventHandler implements ICoscradEventHandler {
+    constructor(
+        @Inject(AUDIO_QUERY_REPOSITORY_TOKEN)
+        private readonly queryRepository: IAudioItemQueryRepository
+    ) {}
+
+    async handle({
+        payload: {
+            text,
+            languageCode,
+            aggregateCompositeIdentifier: { id: audioItemId },
+        },
+    }: AudioItemNameTranslated): Promise<void> {
+        await this.queryRepository.translateName(audioItemId, {
```

Comment:
Note that on the next story, you'll have to write the test coverage and add a new query repository  method. This one just happened to already be done for you this go around.